userspace/libsinsp/cri.h

```diff
@@ -37,9 +37,8 @@ namespace libsinsp {
 namespace cri {
 
 // these shouldn't be globals but we still need references to *the* CRI runtime
-extern std::string s_cri_unix_socket_path;
+extern std::vector<std::string> s_cri_unix_socket_paths;
 extern int64_t s_cri_timeout;
-extern sinsp_container_type s_cri_runtime_type;
```

Comment:
Can we *not* drop these two? `s_cri_unix_socket_path` should point to the one socket path we ended up using.

Reply:
@gnosek it should be ok now.